backend/routes/events.js

```diff
@@ -159,9 +175,15 @@ router.post('/add', async (req, res, next) => {
                 location: processInput(req.body.location, 'hardVC', 'location', 200),
                 instructor: processInput(req.body.instructor, 'hardVC', 'instructor', 100),
                 numberOfVolunteers: processInput(req.body.numberOfVolunteers, 'idNum', 'number of volunteers'),
-                materialsUrl: processInput(req.body.materialsUrl, 'softVC', 'materials url')
+                materialsUrl: processInput(req.body.materialsUrl, 'softVC', 'materials url'),
+                eventDuration: processInput(req.body.eventDuration, 'idNum', 'event duration')
             }
     
+            const calcEventTime = calcHours(eventData.start, eventData.end);
+            if (calcEventTime < calcEventTime.eventDuration) {
```

Comment:
this should be:
if (calcEventTime < **eventData**.eventDuration) {

Reply:
Good catch, thank you